src/python/dxpy/utils/executable_unbuilder.py

```diff
@@ -140,46 +151,60 @@ def make_cluster_bootstrap_script_file(region, entry_point, code, suffix):
     # to distinguish it from non assets. It will be needed to annotate the bundleDepends,
     # when the wrapper record object is no more accessible.
 
-    for dep in reversed(info["runSpec"]["bundledDepends"]):
-        file_handle = get_handler(dep["id"])
-        if isinstance(file_handle, dxpy.DXFile):
-            asset_record_id = file_handle.get_properties().get("AssetBundle")
-            asset_record = None
-            if asset_record_id:
-                asset_record = dxpy.DXRecord(asset_record_id)
-                if asset_record:
+    for region in info["runSpec"]["bundledDependsByRegion"]:
+        for dep in reversed(info["runSpec"]["bundledDependsByRegion"][region]):
+            file_handle = get_handler(dep["id"])
+            if isinstance(file_handle, dxpy.DXFile):
+                asset_record_id = file_handle.get_properties().get("AssetBundle")
+                if asset_record_id:
                     try:
-                        asset_json = {"name": asset_record.describe().get("name"),
-                                              "project": asset_record.get_proj_id(),
-                                              "folder": asset_record.describe().get("folder"),
-                                              "version": asset_record.describe(fields={"properties": True}
-                                                                               )["properties"]["version"]
-                                              }
+                        # asset_desc = dxpy.api.record_describe(asset_record_id, {"properties": True})
+                        # asset_json = {"name": asset_desc.get("name"),
+                        #             "project": asset_desc.get("project"),
+                        #             "folder": asset_desc.get("folder"),
+                        #             "version": asset_desc.get("properties", {}).get("version")
+                        #             }
+                        asset_json={"id": asset_record_id}
```

Comment:
Nice, thanks. Does it work when you "dx get app-swiss-army-knife"? Currently it's pointing to the project ID which I have selected (instead of the record's project) so using record ID should fix it.